### ISSUE TITLE
[Feature Request] Easily displayed map information/notation.

### ISSUE THREAD

**Issue by Mawntee** [Kylemc1413/SongCore#123]
Here's Twitter thread where I first talked about this: https://twitter.com/mawntee/status/1689892916989140992?s=20

TL;DR

Have a basic one or two line text box above the difficulty/playstyle selection (above standard/360/lawless selection, same box style) where the mapper can write some information about the chart. 
Each difficulty can have it's own description, and depending on which dif is selected, the text will change. 

Example: On an Expert plus the mapper can write something like "RT 500 - Resets - Bursts - Pauls" on the Ex, then on the Expert dif they could put something like "RT 550 - Wrist Rolls - Sliders - Poodles" to indicate the difference in downmapping.

This can get rid of a lot of confusion when jumping into a map for the first time, or just act as simple reminders when people are trying to figure out which dif has what.

**Comment by Meivyn** [comment]
The problem with your suggestion is that the UI is already pretty much cluttered currently.

![image](https://github.com/Kylemc1413/SongCore/assets/793322/10b92596-6ae4-44bd-9b75-13649f10d418)

There's not much we can do about this without conflicting with other mods.

**Comment by Mawntee** [comment]
hear me out for a sec...
![Untitled](https://github.com/Kylemc1413/SongCore/assets/51247415/a2d1b1d4-462e-4bd3-afc1-fee624f9be0c)

Slide between song info, warnings, suggestions, etc etc.
Clicking on it will open up a larger window overlapping that can better display things, or hold buttons for extra mods if something else similar to Cinema rolls along in the probably very near future (maybe)

**Comment by Meivyn** [comment]
I like your idea but that is going to be painful to implement from a technical point of view. I can't guarantee your anything and can't give you any ETA but I'll see what I can do.
